lefthk/src/config.rs

```diff
@@ -42,101 +42,100 @@ macro_rules! get_keys {
 #[derive(Debug, Serialize, Deserialize, PartialEq, Clone, Eq)]
 pub struct Keybind {
     pub command: Command,
-    pub modifier: Vec<String>,
+    pub modifier: Option<Vec<String>>,
     pub key: Key,
 }
 
-impl TryFrom<Keybind> for Vec<core_keybind> {
-    type Error = LeftError;
-
-    fn try_from(kb: Keybind) -> Result<Self> {
-        let command_key_pairs: Vec<(core_command, String)> = match kb.command {
-            Command::Chord(children) if !children.is_empty() => {
-                let key = get_key!(kb.key);
-                let children = children
-                    .iter()
-                    .filter_map(|kb| match TryFrom::try_from(kb.clone()) {
-                        Ok(keybinds) => Some::<Vec<lefthk_core::config::Keybind>>(keybinds),
-                        Err(err) => {
-                            tracing::error!("Invalid key binding: {}\n{:?}", err, kb);
-                            None
-                        }
-                    })
-                    .flatten()
-                    .collect();
+fn try_from(kb: Keybind, default_modifier: Vec<String>) -> Result<Vec<core_keybind>> {
```

Comment:
I'd rename this to `try_convert_to_core_keybinds` or something like that

Reply:
This one here, I like it but I don't want to give it a very long name

Reply:
@TornaxO7 what do you think about `try_to_core_keybinds`?

Reply:
understandable. Hm.... In my opinion this big function needs a refactoring anyhow. But this can be done in another PR.

Reply:
I think the same, if you need help, I am available to collaborate

Reply:
> @TornaxO7 what do you think about `try_to_core_keybinds`?

yah! That sonunds better

Reply:
@SergioRibera I started to create a [PR](https://github.com/leftwm/lefthk/pull/29) for this.

Reply:
Great, let me see how that PR goes and see what I can do to help